src/org/labkey/remoteapi/security/RenameContainerCommand.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2009 LabKey Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.labkey.remoteapi.security;
+
+import org.labkey.remoteapi.PostCommand;
+
+import org.json.JSONObject;
+
+/**
+ * Rename a container on the server
+ */
+public class RenameContainerCommand extends PostCommand<RenameContainerResponse> {
+    private String _name;
+    private String _title;
+    private boolean _addAlias = true;
+
+
+    public RenameContainerCommand(String name, String title, boolean addAlias)
+    {
+        super("core", "RenameContainer");
+        _name = name;
+        _title = title;
+        _addAlias = addAlias;
+    }
+
+    @Override
+    public JSONObject getJsonObject()
+    {
+        JSONObject result = new JSONObject();
+        result.put("name", _name);
+        result.put("title", _title);
+        result.put("addAlias", _addAlias);
+
+        return result;
+    }
+
+    public String get_name() {
+        return _name;
+    }
+
+    public void set_name(String _name) {
+        this._name = _name;
+    }
```

Comment:
IntelliJ may have generated these getter and setter names by default, but our usual pattern for all getters and setters is `getName()` and `setName(String name)`. Then `_name = name` (no need for `this._name`).

Reply:
Indeed these were intelliJ generated—lots of weirdness going on, regenerating the getters + setters reverted to our usual pattern. Thanks for the catches Adam!

Reply:
I think a builder pattern to set the values similar to some of the other commands would work well.

Reply:
Ian and I talked about this offline and decided to leave it as-is currently